src/main/java/com/stablekernel/standardlib/BlockingProgressFragment.java

```diff
@@ -28,6 +28,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         Dialog dialog = super.onCreateDialog(savedInstanceState);
         dialog.getWindow().requestFeature(Window.FEATURE_NO_TITLE);
         dialog.getWindow().setBackgroundDrawable(new ColorDrawable(android.graphics.Color.TRANSPARENT));
+        setCancelable(false);
```

Comment:
Yes. I thought about making it optional, but I don't recall it ever being used outside of the scope of a web request and it is just awkward for a user to tap on the screen and dismiss it mid-request.